server/routes/v1/deployment.ts

```diff
@@ -151,6 +152,98 @@ export const postDeployment = [
   },
 ]
 
+export const postUngovernedDeployment = [
+  ensureUserRole('user'),
+  bodyParser.json(),
+  async (req: Request, res: Response) => {
+    req.log.info({ code: 'requesting_deployment' }, 'User requesting deployment')
+    const { body } = req
+
+    body.timeStamp = new Date().toISOString()
+
+    const model = await findModelByUuid(req.user, body.modelUuid)
+
+    if (!model) {
+      throw NotFound(
+        { code: 'model_not_found', modelId: body.modelUuid },
+        `Unable to find model with name: '${body.modelUuid}'`
+      )
+    }
+
+    const name = body.name
+      .toLowerCase()
+      .replace(/[^a-z 0-9]/g, '')
+      .replace(/ /g, '-')
+
+    const uuid = `${name}-${nanoid()}`
+
+    const version = await findVersionByName(req.user, model._id, body.initialVersionRequested)
+    if (!version) {
+      throw NotFound(
+        {
+          code: 'version_not_found',
+          modelId: body.modelID,
+          version: body.initialVersionRequested,
+        },
+        `Unable to find version with name: '${body.initialVersionRequested}'`
+      )
+    }
+
+    const versionArray = [version._id]
+
+    const owner = {
+      kind: 'user',
+      id: req.user.id,
+    }
+
+    const deployment = await createDeployment(req.user, {
+      schemaRef: null,
+      uuid,
+
+      versions: versionArray,
+      model: model._id,
+      metadata: {
+        highLevelDetails: {
+          name: body.name,
+          modelID: model.uuid,
+          initialVersionRequested: body.initialVersionRequested,
+        },
+        contacts: {
+          owner: [owner],
+        },
+      },
+
+      owner,
+    })
+
+    deployment.managerApproved = ApprovalStates.Accepted
+    deployment.ungoverned = true
```

Comment:
Why not set these in the `createDeployment` function above?  Alter it to make them optional args.